module/platform/Webots/src/Webots.cpp

```diff
@@ -391,6 +391,15 @@ namespace module::platform {
                 servo.present_position = 0.0;
                 servo.present_speed    = 0.0;
             }
+
+            auto targets = std::make_unique<ServoTargets>();
```

Comment:
Need some comments here and probably change the message name to something more general, Cameron and I had `ResetSimulation` last night, I think that is more accurate.

Reply:
Comments around thing would be good